autogpt/memory/redismem.py

```diff
@@ -10,16 +10,18 @@
 from redis.commands.search.indexDefinition import IndexDefinition, IndexType
 from redis.commands.search.query import Query
 
+from autogpt.config import Config
 from autogpt.llm_utils import create_embedding_with_ada
 from autogpt.logs import logger
 from autogpt.memory.base import MemoryProviderSingleton
 
```

Comment:
Config loading same as above @collijk

Reply:
Same as before, if you saying "use the cfg passed to the class functions", it's not pussible here, i need it in a global variable
```
CFG = Config()
SCHEMA = [
    TextField("data"),
    VectorField(
        "embedding",
        "HNSW",
        {"TYPE": "FLOAT32", "DIM": 1536, "DISTANCE_METRIC": "COSINE"},
        {"TYPE": "FLOAT32", "DIM": CFG.embed_dim, "DISTANCE_METRIC": "COSINE"},
    ),
]
```

If it's not that, i don't know what are you referring to

Reply:
can that global variable be moved to not global?

Reply:
Done